XTS/WS-C/dev/src/com/arjuna/webservices11/wscoor/sei/RegistrationResponsePortTypeImpl.java

```diff
@@ -0,0 +1,63 @@
+ * by the @authors tag.  All rights reserved. 
+ * See the copyright.txt in the distribution for a full listing 
+ * of individual contributors.
+ * This copyrighted material is made available to anyone wishing to use,
+ * modify, copy, or redistribute it subject to the terms and conditions
+ * of the GNU Lesser General Public License, v. 2.1.
+ * This program is distributed in the hope that it will be useful, but WITHOUT A
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License,
+ * v.2.1 along with this distribution; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA  02110-1301, USA.
+ * 
+ * (C) 2005-2006,
+ * @author JBoss Inc.
+ */
+package com.arjuna.webservices11.wscoor.sei;
+
+import com.arjuna.webservices.logging.WSCLogger;
+import com.arjuna.webservices11.wsaddr.AddressingHelper;
+import com.arjuna.wsc11.AsynchronousRegistrationMapper;
+import javax.jws.*;
+import javax.jws.soap.SOAPBinding;
+import javax.xml.ws.Action;
+import javax.xml.ws.WebServiceContext;
+import javax.xml.ws.soap.Addressing;
+import javax.annotation.Resource;
+import javax.xml.ws.handler.MessageContext;
+import org.jboss.ws.api.addressing.MAP;
+
+@WebService(targetNamespace = "http://docs.oasis-open.org/ws-tx/wscoor/2006/06", 
+        name = "RegistrationResponsePortType",
+        serviceName = "RegistrationResponseService",
+        portName = "RegistrationResponsePortType"
+)
+@SOAPBinding(parameterStyle = SOAPBinding.ParameterStyle.BARE)
+@HandlerChain(file="/ws-c_handlers.xml")
+@Addressing(required=true)
+public class RegistrationResponsePortTypeImpl {
+    
+    @Resource private WebServiceContext webServiceCtx;
+
+    @WebMethod(operationName = "RegisterResponseOperation", action = "http://docs.oasis-open.org/ws-tx/wscoor/2006/06/RegisterResponse")
+    @Oneway
+    @Action(input="http://docs.oasis-open.org/ws-tx/wscoor/2006/06/RegisterResponse")
+    public void registerResponseOperation(
+        @WebParam(targetNamespace = "http://docs.oasis-open.org/ws-tx/wscoor/2006/06", partName = "parameters", name = "RegisterResponse")
+        org.oasis_open.docs.ws_tx.wscoor._2006._06.RegisterResponseType parameters) {
+        
+        MessageContext ctx = webServiceCtx.getMessageContext();
+        MAP inboundMap = AddressingHelper.inboundMap(ctx);
+        if (inboundMap.getRelatesTo() != null)  {
+            WSCLogger.logger.tracev("RegistrationResponsePortTypeImpl received resposne for messageId {0}", inboundMap.getRelatesTo().getRelatesTo());
+            AsynchronousRegistrationMapper.getInstance().assignResponse(inboundMap.getRelatesTo().getRelatesTo(), parameters);
+        } else {
+            WSCLogger.logger.error("RegistrationResponsePortTypeImpl message received for unknown messageId");
```

Comment:
because the `warn` and `error` messages should be translated then we have a rule that they have to be created in the https://github.com/jbosstm/narayana/blob/master/XTS/WS-C/dev/src/com/arjuna/webservices/logging/wscI18NLogger.java#L304 and then the logger method is called here.

Could you please change all the `error` and `warn` messages to be defined under `wscI18NLogger`?